### PR TITLE
first commit for homepage

### PR DESCRIPTION
**Pull request by madhewett**

### CODE REVIEW

File: html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <h1>Artist Biography</h1>
+
+<style>
+</style>
```
Comment (nbriz) [inline html]: the ```<style> </style>``` element is used to write CSS directly in ur HTML ( as opposed to writing the CSS in it's own file && linking to it w/the ```<link>``` element ), that said, u've got ur CSS linked via the ```<link>``` element, so u don't need these tags at all... so i would delete these two lines.

File: html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+<link rel="stylesheet" href=cp-1022_reasearch-studio-II/style.css>
```
Comment (nbriz) [inline html]: this link tag is pointing to a CSS file inside of a folder called "cp-1022_reasearch-studio-II", but the *style.css* file is actually in the same folder as this html file, so what u actually need to write is: 
```html
<link rel="stylesheet" href="style.css">
```
also notice that i added quotes around the file name (ur link tag above is missing the attribute value's quotes)

File: html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+  </head>
+  <body>
+  <h3>By Madelynne Hewett</h3>
+
+<div class="content"
+  </body>
```
Comment (nbriz) [inline html]: u have 2 closing ```</body>``` tags, one here && another at the bottom of ur file on line 36. all ur content elements like ```<div>, <p>, etc.``` should be inside ur body, meaning they should all be between the opening ```<body>``` tag && the closing ```</body>``` tag. which means u should delete this tag on line 16 && keep the one on line 36

File: style.css [hunk 4]
```diff
@@ -0,0 +1,83 @@
+<style>
```
Comment (nbriz) [inline style.css]: this is a CSS file ( hence the file extention, styles.*css* ), which means u should only be writing CSS code in here ( not HTML code ), the ```<style>``` element is HTML && should only be used in ur HTML page ( specifically it's used to write CSS code inside an HTML page ). so u should delete this line as well as the closing ```</style>``` tag at the bottom of this file

**Comment by madhewett** [comment]
Thank you for the edits, I’ll have these corrected by tonight. 

-Mady

**Comment by nbriz** [comment]
thnx for these changes Maddie, except u still need to change the name of ur HTML page, i mentioned before:

> first u need to rename ur html page... right now it's simply "html" it should be "index.html" ( once u change that this link https://madhewett.github.io/ will go to ur homepage instead of ur README ).

**Comment by madhewett** [comment]
Got it, thank you! 

-Mady

**Pull request merged**